algebird-test/src/test/scala/com/twitter/algebird/MaxLaws.scala

```diff
@@ -0,0 +1,53 @@
+package com.twitter.algebird
+
+import com.twitter.algebird.BaseProperties._
+import com.twitter.algebird.scalacheck.arbitrary._
+import com.twitter.algebird.scalacheck.NonEmptyVector
+import org.scalacheck.Arbitrary
+import org.scalacheck.Prop.forAll
+
+class MaxLaws extends CheckProperties {
+  def maxTest[T: Arbitrary: Ordering] =
+    forAll { (l: Max[T], r: Max[T]) =>
+      val realMax = Max(Ordering[T].max(l.get, r.get))
+      l + r == realMax && (l max r) == realMax
+    }
+
+  property("Max.{ +, max } works on ints") { maxTest[Int] }
+
+  property("Max should work on non-monoid types like String") { maxTest[String] }
```

Comment:
good catch - I was using the same tests in `MinLaws` and `MaxLaws`. I'll change types. I was looking to exercise a semigroup, since it's defined differently from the monoid